pkg/prometheus/operator.go

```diff
@@ -1037,9 +1058,59 @@ func (c *Operator) createTPRs() error {
 	}
 
 	// We have to wait for the TPRs to be ready. Otherwise the initial watch may fail.
```

Comment:
This is the createTPRs function, the createCRDs function has its appropriate name.